doc/src/devdocs/probes.md

```diff
@@ -90,10 +90,24 @@ the probe handler.
 3. `julia:gc__mark__begin`: Beginning the mark phase
 4. `julia:gc__mark_end(scanned_bytes, perm_scanned)`: Mark phase ended
 5. `julia:gc__sweep_begin(full)`: Starting sweep
-6. `julia:gc__sweep_end()`: Sweep phase finished
+6. `julia:gc__sweep_end`: Sweep phase finished
 7. `julia:gc__end`: GC is finished, other threads continue work
 8. `julia:gc__finalizer`: Initial GC thread has finished running finalizers
 
+### Task runtime probes
+
+1. `julia:rt__run__task(task, ptls)`: Switching to task `task` on thread associated with PTLS `ptls`.
+2. `julia:rt__pause__task(task, ptls)`: Switching from task `task` on thread associated with PTLS `ptls`.
+3. `julia:rt__new__task(parent, child, ptls)`: Task `parent` created task `child` on thread associated with PTLS `ptls`.
+4. `julia:rt__start__process__events(task)`: Task `task` started processing libuv events.
+5. `julia:rt__finish__process__events(task)`: Task `task` finished processing libuv events.
+6. `julia:rt__multiq__insert__success(ptls)`: Thread with PTLS `ptls` successfully inserted a task.
+7. `julia:rt__multiq__insert__full(ptls)`: Thread with PTLS `ptls` failed to insert a task because the multiq was full.
+8. `julia:rt__multiq__deletemin__trylock__success(ptls, heap)`: Thread with PTLS `ptls` successfully acquired the lock for multiq heap `heap`.
+9. `julia:rt__multiq__deletemin__trylock__failed(ptls, heap)`: Thread with PTLS `ptls` failed to acquire a lock.
```

Comment:
Maybe something like

```suggestion
9. `julia:rt__multiq__deletemin__trylock__failed(ptls, heap)`: Thread with PTLS `ptls` failed to acquire a lock. If the number of failed lock attempts significantly exceeds the number of successful locks, it indicates the contention on the (re)scheduling of non-sticky tasks.
```

?

Reply:
Could we also have a bpftrace script that uses these multiqueue probes? And some docs in the uprobes chapter that show how to use it?